src/constants/wizardForm.js

```diff
@@ -0,0 +1,20 @@
+const interestsList = [
+  { id: 1, name: 'Sport' },
```

Comment:
What if you want to concat several arrays? Ids have to be unique. As you might want to dynamically translate them, it is better to use ids like:

`id: 'formListInterestsSport',`

then you will reuse it in translation files, and the id will be unique.
Also it is better for future to use immutable Map:

```
const formListInterestsSportId = 'formListInterestsSport';
const interestsList = Map(
{
[formListInterestsSportId ]: { id: formListInterestsSportId },
...
}
)
```

Then you can convert it to array, use it like object, do a lot of things with this stuff... Very convenient in future.